### DISCUSSION TITLE
can't navigate and modify in (run and debug view) console output tab

### DISCUSSION THREAD

**Discussion by lsw9521** [JetBrains/ideavim#425] [Q&A]
Can't navigate and modify in (run and debug view) console output tab. You can't even enter normal mode in the console by esc ! It greatly affects the fluency of operation！I found it appears since the version (0.69) .  So I'm still using this old version of 0.65！

**Comment by paper-tiger** [comment]
I have the same problem today. Workaround: Select some text in the output with the mouse then it suddenly works.

**Comment by citizenmatt** [comment]
This should be fixed as part of IdeaVim 2.5.0

**Comment by Thogor** [comment]
There seems to be a regression because this bug popped up again

**Reply by Thogor** [reply]
Also, the mouse workaround doesn't work anymore

**Comment by piacsek** [comment]
I've experienced the same thing, so I created [another plugin](https://github.com/piacsek/test-output-selector) to allow me to jump into the console straight in VIM mode 👇 

https://github.com/user-attachments/assets/af3a81bc-a2af-4b0c-8077-718b6eb6ebcf


After installing the custom plugin from disk, I added this to my `.ideavimrc`:

```
map <leader>9 <Action>(com.testoutputselector.FocusTestRunnerAction)
```
